### PR TITLE
Zip/Unzip support up to 22 ports

### PR DESCRIPTION
**Pull request by jchapuis**
As discussed with @patriknw and @johanandren, here's a PR that corrects the small inconsistency in terms of number of ports for Zip and Unzip graphstages

### CODE REVIEW

**Comment by johanandren** [comment]
OK TO TEST

**Comment by johanandren** [comment]
Test failure seems related: `akka.stream.scaladsl.GraphUnzipWithSpec.UnzipWith must work with up to 22 outputs`

**Comment by jchapuis** [comment]
sorry, it was a small blunder (and I forgot that `testOnly` was case-sensitive when running `testOnly *Zip*` 😉)

**Comment by jchapuis** [comment]
mima exclusions done

**Review by johanandren — APPROVED** [review]
LGTM

File: akka-stream/src/main/mima-filters/2.5.x.backwards.excludes [hunk 1]
```diff
@@ -60,4 +60,8 @@ ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowO
 ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowOps.zipLatestGraph")
 ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowOps.zipLatest")
 ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowOps.zipLatestWithGraph")
-ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowOps.zipLatestWith")
\ No newline at end of file
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowOps.zipLatestWith")
+
+# zip/unzip 22 parameters support
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.ZipWithApply.apply")
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.UnzipWithApply.apply")
```
Comment (patriknw) [inline akka-stream/src/main/mima-filters/2.5.x.backwards.excludes]: I think this should be added to the `akka-stream/src/main/mima-filters/2.5.17.backwards.excludes` instead

**Comment by jchapuis** [comment]
ok done

**Review by chbatey — APPROVED** [review]
LGTM

**Pull request merged**